appsec/src/extension/ddappsec.c

```diff
@@ -451,6 +457,46 @@ static PHP_FUNCTION(datadog_appsec_testing_request_exec)
     RETURN_TRUE;
 }
 
+static PHP_FUNCTION(datadog_appsec_push_params)
```

Comment:
You should make this generic and provide functionality to pass one or even many addresses to the WAF, of arbitrary type.

Reply:
Done